src/main/java/io/kontur/disasterninja/service/layers/providers/LayersApiProvider.java

```diff
@@ -57,15 +55,12 @@ public CompletableFuture<List<Layer>> obtainSelectedAreaLayers(LayerSearchParams
 
     @Override
     public Layer obtainLayer(String layerId, LayerSearchParams searchParams) {
-        if (!isApplicable(layerId)) {
-            return null;
-        }
         return layersApiClient.getLayer(searchParams.getBoundary(), layerId, searchParams.getAppId());
     }
 
     @Override
     public boolean isApplicable(String layerId) {
-        return layerId.startsWith(LAYER_PREFIX);
+        return true;
     }
```

Comment:
why do you need a function that always returns true? just curious 😄

Reply:
To define which layer provider to use in order to load a layer
https://github.com/konturio/disaster-ninja-be/blob/main/src/main/java/io/kontur/disasterninja/service/layers/LayerService.java#L167